unstructured_ingest/v2/processes/connectors/fsspec/box.py

```diff
@@ -34,7 +35,7 @@ class BoxIndexerConfig(FsspecIndexerConfig):
 
 class BoxAccessConfig(FsspecAccessConfig):
     box_app_config: Optional[str] = Field(
-        default=None, description="Path to Box app credentials as json file."
+        default=None, description="Box app credentials as a JSON string."
```

Comment:
Can use this utility to leveraeg pydantic to do the validation: [google_drive](https://github.com/Unstructured-IO/unstructured-ingest/blob/main/unstructured_ingest/v2/processes/connectors/google_drive.py#L41)

Reply:
Good point, fixed!